spi/imx6ull-qspi/qspi.c

```diff
@@ -0,0 +1,326 @@
+/*
+ * Phoenix-RTOS
+ *
+ * i.MX 6ULL QSPI lib
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Hubert Badocha
+ *
+ * %LICENSE%
+ */
+
+#include <stdio.h>
+#include <stddef.h>
+#include <sys/interrupt.h>
+#include <sys/mman.h>
+#include <errno.h>
+#include <sys/platform.h>
+#include <sys/threads.h>
+#include <stdbool.h>
+#include <string.h>
+#include <phoenix/arch/imx6ull.h>
+#include "qspi.h"
+#include <board_config.h>
```

Comment:
Just an OCD thing, so local headers are not mixed with global ones

```suggestion
#include <board_config.h>
#include "qspi.h"
```